### PR TITLE
impr(profile/account page): show leaderboard rank percentage (fehmer)

### PR DESCRIPTION
**Pull request by fehmer**
### Description

Add rank percentage of the all time leaderboards to the account and profile page:

image

### CODE REVIEW

**Comment by Miodec** [comment]
I would place it under the "15/60 seconds"

File: frontend/src/ts/constants/default-snapshot.ts
Comment (ruru-m07) [inline frontend/src/ts/constants/default-snapshot.ts:1]: just formatting code.

**Review by ruru-m07 — APPROVED** [review]
all changes looks ok.

**Review by ruru-m07 — APPROVED** [review]
all changes looks ok.

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/8343636601

**Comment by Miodec** [comment]
Thanks

**Review by ruru-m07 — COMMENTED** [review]
NICE WORK 👍

**Pull request merged**